src/devices/vdml_motors.cpp

```diff
@@ -215,8 +215,8 @@ std::vector<pros::v5::Motor_Brake> Motor_Group::get_every_brake_mode(void) const
 	std::vector<pros::v5::Motor_Brake> return_vector;
 	empty_motor_group_check_vector(pros::v5::Motor_Brake::invalid, return_vector);
 	push_motor_configuration();
-	for (int i = 0; i < _ports.size(); i++) {
-		return_vector.emplace_back(static_cast<pros::v5::Motor_Brake>(motor_get_brake_mode(_ports[i])));
+	for (auto it = _ports.begin() + 1; it < _ports.end(); it++) {
```

Comment:
the +1 in the initializer must be removed here, it's only for if you're explicitly performing the action on the first element outside the loop, like in `brake()`.  Same applies to most of the functions below

Reply:
Good catch. I missed this when I used replace all at midnight